### DISCUSSION TITLE
Oracle Wallet trino Config ???

### DISCUSSION THREAD

**Discussion by eadapa29** [trinodb/trino#13385] [General]
Hello, 
   I am trying to connect to Oracle via Oracle wallet. Does trino Oracle connector support that config ?
 I only have a Oracle.sso file... Wondering what would be the JDBC Connection string would look like ?

regards,
Eshwara

**Comment by ebyhr** [comment]
Did you try `TNS_ADMIN` option?
https://docs.oracle.com/en/cloud/paas/autonomous-database/adbsa/connect-jdbc-thin-wallet.html#GUID-1640CC02-BF3E-48C2-8FFE-A596614A6A40

**Comment by eadapa29** [comment]
TNS_ADMIN option is not clear to me but based on the documentation wallet
means we need to have mutual TLS ?

**Comment by ebyhr** [comment]
I'm not sure. I would recommend accessing without Trino first (e.g. using DBeaver) and then copying the connection string to properties file.

**Comment by aakashnand** [comment]
@eadapa29 Were you able to solve this?
